src/models/userDao.js

```diff
@@ -0,0 +1,98 @@
+const { sqlDataSource } = require("./data-source");
+
+const getUserbyEmail = async (email) => {
+  const [user] = await sqlDataSource.query(
+    `
+        SELECT
+            id,
+            nickname,
+            email,
+            social_id
+        FROM 
+          users
+        WHERE 
+          users.email = ?
+        `,
+    [email]
+  );
+  return user;
+};
+const createUser = async (nickname, email, id) => {
+  try {
+    const users = await sqlDataSource.query(
+      `INSERT INTO users(
+          nickname,
+          email,
+          social_id
+          ) 
+        VALUES (?, ?, ?)
+        `,
+      [nickname, email, id]
+    );
+    return users;
+  } catch (err) {
+    console.log(err);
+    throw new Error("INVALID DATA INPUT");
+  }
+};
+const addUserInfo = async (kakaoId, name, phone, date) => {
```

Comment:
```suggestion
const updateUserInfo = async (kakaoId, name, phone, date) => {
```
update라는 keyword가 조금 더 명확하게 의미를 전달할 수 있을 것 같아요~